### ISSUE TITLE
[Bug] Xamarin.Forms.Platform.iOS.EntryRendererBase`1[TControl].GetClearButtonTintImage crash

### ISSUE THREAD

**Issue by AndreasReitberger** [xamarin/Xamarin.Forms#14510]
### Description
Using iOS 13 on an iPhone 8 simulator leads to following crash while using entry box with ClearButtonVisibility to set "WhileEditing".

```xaml
<Entry 
      Grid.Column="1"
      Placeholder="{x:Static localization:Strings.SettingsPrintServerConnectionAddress}"
      Text="{Binding ServerAddress, Mode=TwoWay}" 
      Style="{StaticResource DefaultBorderlessEntry}"
      ClearButtonVisibility="WhileEditing"
      Margin="16,2,16,4"
      />
```
Newer iOS versions are working fine.

### Steps to Reproduce

1. Setup an iPhone 8 simulator running iOS 13.1
2. Insert the xaml code above
3. See the exception

### Expected Behavior
No exception.

### Actual Behavior
Exception gets thrown.
`	exc	{System.NullReferenceException: Object reference not set to an instance of an object   at Xamarin.Forms.Platform.iOS.EntryRendererBase`1[TControl].GetClearButtonTintImage (UIKit.UIImage image, UIKit.UIColor color) [0x00000] in D:\a\1\s\Xamarin.Forms.Platform.iO…}	
`
### Basic Information

- Version with issue: SR4
- Platform Target Frameworks: 
  - iOS:  iOS 13.1 on iPhone 8 (Simulator and real device)
- Affected Devices: iPhone 8 (Simulator / real device)

### Stacktrace

Xamarin.Forms.Platform.iOS.EntryRendererBase`1[TControl].GetClearButtonTintImage (UIKit.UIImage image, UIKit.UIColor color) [0x00000] in D:\\a\\1\\s\\Xamarin.Forms.Platform.iOS\\Renderers\\EntryRenderer.cs:591 \n at Xamarin.Forms.Platform.iOS.EntryRendererBase`…"

**Comment by AndreasReitberger** [comment]
Seems to be related to: https://github.com/xamarin/Xamarin.Forms/issues/14479

**Comment by jfversluis** [comment]
Fixed with #14479
